Sources/Datadog/Core/Feature.swift

```diff
@@ -120,6 +120,19 @@ internal struct FeatureStorage {
         self.reader = reader
         self.arbitraryAuthorizedWriter = arbitraryAuthorizedWriter
     }
+
+#if DD_SDK_COMPILED_FOR_TESTING
+    /// Flushes all async write operations.
+    ///
+    /// This method is executed synchronously. After return, the storage feature has no more
+    /// pending asynchronous write operations so all its data is ready for upload.
+    func flush() {
```

Comment:
`flush()` is not `public`, why do we wrap it with `#if DD_... #endif`?

Reply:
To avoid using `flush()` and `deinitialize()` APIs in production code. These features are highly experimental and work with few arbitrary assumptions, so using them for serving any other logic in production code would be dangerous.